app/models/prisoner.rb

```diff
@@ -8,4 +8,14 @@ class Prisoner < ActiveRecord::Base
   validates :surname, presence: true
   validates :date_of_birth, presence: true
   validates :gender, presence: true
+
+  def self.search(query=[])
+    return [] if query.nil?
```

Comment:
I think if you return self.none here you have a query that follows the AR api a little closer. As in it would be chainable and be the same as performing the where below and not finding anything. Sorry for PR snatching in advance, just looking through thr code.